processing/src/main/java/org/apache/carbondata/processing/sort/sortdata/IntermediateSortTempRowComparator.java

```diff
@@ -36,13 +38,35 @@
 
   private DataType[] noDicSortDataTypes;
 
+  private Map<DataType, SerializableComparator> comparator_map;
+
   /**
    * @param isSortColumnNoDictionary isSortColumnNoDictionary
    */
   public IntermediateSortTempRowComparator(boolean[] isSortColumnNoDictionary,
       DataType[] noDicSortDataTypes) {
     this.isSortColumnNoDictionary = isSortColumnNoDictionary;
     this.noDicSortDataTypes = noDicSortDataTypes;
+    comparator_map = new HashMap<>();
```

Comment:
Now this code is not required..Please remove

Reply:
removed